<launch>
  <arg name="openni" default="true"/>
  <arg name="kinect" default="false"/>
  <arg name="xtion" default="true"/>
  <arg name="publish_tf" default="true"/>

  <include if="$(arg openni)" file="$(find pandora_slam_3d)/launch/openni.launch">
    <arg name="kinect" value="$(arg kinect)"/>
    <arg name="xtion" value="$(arg xtion)"/>
    <arg name="publish_tf" value="$(arg publish_tf)"/>
  </include>

  <node
    name="point_cloud_subsampler"
    type="point_cloud_subsampler"
    pkg="pandora_slam_3d"
    output="screen"/>
</launch>
